marurevi/Ruby-capstone
# Fix book class

## Add new parameter for Book class
- Fix book.rb file
- Implement the changes on app.rb file
- Modify book_spec.rb file to test the new parameter
- run rubocop to fix linters
- check the correct setup of the aplication
